src/emulator/constants.ts

```diff
@@ -13,6 +13,7 @@ const DEFAULT_HOST = "localhost";
 
 export class Constants {
   static SERVICE_FIRESTORE = "firestore.googleapis.com";
+  static SERVICE_REALTIME_DATABASE = "firebaseio.com";
```

Comment:
Let's find out the service used for production and go with that. Maybe we can get this from the functions SDK, or maybe we can test this in prod.

Reply:
I verified that this gets set to `firebaseio.com` in production.